### PR TITLE
synapse best practice guide

### PR DESCRIPTION
**Pull request by dataders**
## What are you changing in this pull request and why?

as part of launch Synapse in dbt Cloud, here's some best practices (related: #5435)

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

**Review by jasnonaz — CHANGES_REQUESTED** [review]
Per notion - let's make this a developer blog. Let's move over to dev blog format then get ready to roll

File: website/blog/2024-05-17-synapse-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
+---
+title: Up and Running with Azure Synapse on dbt Cloud
+description: "How to get started with Azure Synapse on dbt Cloud"
```
Comment (dataders) [inline website/blog/2024-05-17-synapse-best-practices.md]: @jasnonaz would love your input on `title` and `description`

Comment (jasnonaz) [inline website/blog/2024-05-17-synapse-best-practices.md]: maybe some tips for getting started

**Review by jasnonaz — APPROVED** [review]
Approving with the fixes we made live on call

**Pull request merged**